boards/st/nucleo_l552ze_q/board.yml

```diff
@@ -0,0 +1,7 @@
+board:
+  name: nucleo_l552ze_q
+  vendor: st
+  socs:
+    - name: stm32l552xx
```

Comment:
- name should go left by 2 spaced, then all lines below also

Reply:
This is the way it is generated by `scripts/utils/board_v1_to_v2.py` script.

Reply:
@gmarull

Reply:
our `.yamllint` file says `indentation: spaces: 2`, I guess it would complain if this was wrong.

Reply:
@fabiobaltieri ?

Reply:
from https://github.com/zephyrproject-rtos/zephyr/blob/main/.yamllint

```
  indentation:
    spaces: 2
    indent-sequences: consistent
```

the lists are a bit odd for some reasons, the linter just checks that the stile is coherent throughout the file